src/sentry/lang/native/symbolicator.py

```diff
@@ -142,6 +142,16 @@
 }
 
 
+def _get_worker_id():
+    # generate worker id on first use doing this at import time caused all
+    # celery worker processes to share worker id which is not desired
+    global WORKER_ID
+    if WORKER_ID is None:
+        # % 5000 to reduce cardinality of metrics that we tag with worker_id
+        WORKER_ID = str(uuid.uuid4().int % 5000)
```

Comment:
Do we really need to use a global variable for that? Maybe we could do this initialization inside `SymbolicatorSession` constructor? And use `self.worker_id` where needed.

Reply:
Not happy about the global variable but if we want to keep I wanted to keep it static for the life of worker process.

`SymbolicatorSession` object is created for each `Symbolicator` object which in turn is created in each symbolication function:
 - https://github.com/getsentry/sentry/blob/master/src/sentry/lang/native/processing.py#L257
 - https://github.com/getsentry/sentry/blob/master/src/sentry/lang/native/processing.py#L275
 - https://github.com/getsentry/sentry/blob/master/src/sentry/lang/native/processing.py#L343

I guess we can use class attribute instead global variable. I'll try to change it.